src/main/java/com/xpandit/plugins/xrayjenkins/task/XrayImportBuilder.java

```diff
@@ -701,9 +800,13 @@ private UploadResult tryUploadResults(
             }
 
             listener.getLogger().println("Starting to import results from " + resultsFile.getName());
-
             UploadResult result = client.uploadResults(targetEndpoint, dataParams, queryParams);
-
+            
+            dataParams.values().stream()
+                      .map(Content::getContent)
+                      .filter(content -> content instanceof AutoCloseable)
+                      .forEach(content -> closeAutoCloseableInstances(listener, (AutoCloseable) content));
```

Comment:
if you want, you can call a _.map()_ and cast all the variables, I think it would make the stream a bit cleaner, but I will leave this up to you:
```java
.filter(content -> content instanceof AutoCloseable)
.map(AutoCloseable.class::cast)
.forEach(content -> closeAutoCloseableInstances(listener, content));
```